components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -301,9 +288,9 @@ public function get_pod_filter( $pod_name = '' ) {
 		if ( isset( $possible_pods[ $pod_name ] ) ) {
 			$this_pod = $possible_pods[ $pod_name ];
 
-			if ( in_the_loop() && ! is_singular() ) {
+			if ( is_archive() || is_post_type_archive() || is_tax() ) {
 				$filter = pods_v( 'archive_filter', $this_pod, $filter, true );
-			} else {
+			} elseif ( is_singular() || is_single() ) {
```

Comment:
Why add an `elseif` here? The single filters might be needed on other locations.
Also, current fix doesn't check for `in_the_loop()` and it's very much possible to add archive-like loops in singular pages.

Reply:
Adjusted